src/containers/SearchPage/searchHelpers.js

```diff
@@ -130,6 +131,9 @@ export const convertSearchParam = value => {
   if (Number.isInteger(value)) {
     return value;
   }
+  if (typeof value === 'boolean') {
```

Comment:
Jeg fikk sååååå mye shit fra @jnatten når jeg brukte `typeof` for å sjekke om variabelen er en funksjon type. Da lærte jeg at det å bruke `typeof` ikke er bra.

Reply:
Kom gjerne med et forslag til bedre løsning.

Reply:
Ja nei, kikka litt rundt og ser at det å refactore kode blir for mye arbeid for det det er verdt.